modules/official-blueprints.md

```diff
@@ -37,13 +37,17 @@ Stream lead is: [Daniel Petisme](https://github.com/danielpetisme).
 
 ## Node.js
 
-Latest official blueprint, it replaces the server java side with the NestJS Node framework. Source code and documentation could be found here: [repository](https://github.com/jhipster/generator-jhipster-nodejs).
+Fourth official blueprint, it replaces the server java side with the NestJS Node framework. Source code and documentation could be found here: [repository](https://github.com/jhipster/generator-jhipster-nodejs).
```

Comment:
Numbering these probably isn't a good idea. Will be a maintenance headache and doesn't adding anything. I think just a general description for each is good.

Removing latest is a good idea but i don't think the ordering is needed.

Reply:
I wonder the same, to be honest. We should not add the idea there is a kind of order. I'll remove all this numbers.